tests/unit/s2n_handshake_test.c

```diff
@@ -434,6 +434,42 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Ensure that a handshake can be performed after all file descriptors are closed */
+    {
+        /* A fork is created to ensure that closing file descriptors (like stdout) won't impact
+         * other tests.
+         */
+        pid_t pid = fork();
+        if (pid == 0) {
+            long max_file_descriptors = sysconf(_SC_OPEN_MAX);
+            EXPECT_SUCCESS(max_file_descriptors);
```

Comment:
Wait doesn't EXPECT_SUCCESS check for == 0? Doesn't this mean sysconf is always returning 0?

Reply:
https://github.com/aws/s2n-tls/blob/main/utils/s2n_safety_macros.h#L383 Oh no, we check > S2N_FAILURE, not == S2N_SUCCES. I'd still suggest being more explicit here since this isn't an s2n method.